CHANGELOG.md

```diff
@@ -11,6 +12,48 @@ Version numbers 0.2.x to 0.7.x were intentionally not used to avoid conflicts wi
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+# v0.9.0
+
+**This is the third alpha version of the Quality-On-Demand (QoD) API.**
+
+- [API definition with inline documentation](https://github.com/camaraproject/QualityOnDemand/releases/tag/release-0.9.0/code/API_definitions)
+
+## Please note:
+
+- **This release contains significant breaking changes compared to v0.8.1, and it is not backward compatible**
```

Comment:
Semantic versioning rules for major version 0 are quite clear:
> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

There is no expectation that the major version must be incremented whenever a breaking change is made to a pre-release version. Those rules only apply from v1.0.0 onwards. This should be clarified in the CAMARA versioning rules, which were adopted without any great discussion.

Reply:
Since we access the endpoints with only the major version, this does cause challenges for early development and can limit early adoption / testing prior to releasing an API.

It's a fairly simple change to allow the endpoints to denote which version(s) of the api are compatible with it.

Reply:
I think that is something we need to live with for pre-release APIs. Surely your test clients know what pre-release version you have implemented anyway?

But I think you are mixing two issues. Semantic versioning rules do not say that the major version must be included in the endpoint path - that is just a convention. For a pre-release version with major version 0, there is nothing to stop us having basepaths such as `qod/v0-9-0` or `qod/1Alpha1`. We don't need to bend semantic versioning rules to achieve that. We just need some rule to match the version number to the basepath value.

Reply:
@RandyLevensalor 

You are fully right that the exposed API should have different base paths and I'm in favor of the `/qod/v0alpha3` (as this is the third alpha version of the API) notation as proposed by you in https://github.com/camaraproject/Commonalities/issues/36. But I don't see a justification for /qod/v1Alpha1.9.0 as the release number shouldn't be in the path as long as the releases are backward compatible and the client does not need to change. We need another mechanism to determine the exact release by the client.

Also /qod/v09 would work for now, but not any more if we have more than one API with different alpha, beta or stable stati within the sub project (remember that v0.9.0 is the release number of the complete repository).

And I agree that we could have used /qod/**v1**alpha1 already for the v0.8.0 release (the first alpha release of what will eventually get the stable v1 version of the API). But given the huge changes, I'm quite happy we haven't done that.

BTW: the base path is a variable and /qod/v0 is just the default value. So it is compliant with the definition to use one of the notations in the base path of your implementation to distinct the pre-stable APIs from each other for your API consumers:
```

     basePath:
        default: qod/v0
        description: Base path for the QoD API

```

Reply:
@RandyLevensalor @eric-murray I propose to keep the base path default for now as is, and engage within Commonalities for a common rule to define the base path for pre-stable (not "pre-release") API versions which we then apply to the next release.